Makefile

```diff
@@ -1,4 +1,9 @@
-.PHONY: clean test verbose coverage 
+.PHONY: all clean test verbose coverage 
+
+all:
+	cd data && make data 
+	cd data && make validate 
+	cd final && make all 
 
```

Comment:
Didn't run this one since make data = downloading 6 GB (tried running a phony Makefile recipe with the testing data instead), but yes, you can cd into other directories and call make commands defined in them.